src/sdg_hub/blocks/llmblock.py

```
@@ -278,7 +355,21 @@ def _format_prompt(self, sample: Dict) -> str:
 
         return self.prompt_template.render(**sample).strip()
 
-    def _validate(self, prompt_template: str, input_dict: Dict[str, Any]) -> bool:
+    def _validate(self, prompt_template: Union[str, Template], input_dict: Dict[str, Any]) -> bool:
```

Pylint (3.3.7)

[warning] Number of parameters was 2 in 'Block._validate' and is now 3 in overriding 'ConditionalLLMBlock._validate' method

(W0221)